fenix/app/src/main/java/org/mozilla/fenix/home/TabCounterView.kt

```diff
@@ -27,30 +26,32 @@ import org.mozilla.fenix.tabstray.Page
  * Helper class for building the [FenixTabCounterMenu].
  *
  * @param context An Android [Context].
- * @param browsingModeManager [BrowsingModeManager] used for fetching the current browsing mode.
  * @param navController [NavController] used for navigation.
  * @param tabCounter The [TabCounter] that will be setup with event handlers.
+ * @param mode The current [BrowsingMode].
+ * @param itemTapped Callback to update the [BrowsingMode].
```

Comment:
```suggestion
 * @param onBrowsingModeChange Callback to update the [BrowsingMode].
```
Maybe `onBrowsingModeChange` to keep with how we name our lambdas